storage/Cargo.toml

```diff
@@ -35,10 +35,10 @@ bytemuck_derive = "1.10.0"
 enum-as-inner = "0.6.1"
 integer-encoding = "4.0.2"
 lru = "0.16.0"
-nonzero_ext = "0.3.0"
```

Comment:
fyi: your merge conflict resolution removed the deps here and in `fwdctl`.

Reply:
I don't think we need it here now that we have moved on with the new `LinearAddress` type

Reply:
yeah, I noticed CI is passing but there is a reference to it in fwdctl visible in the diff context here

https://github.com/ava-labs/firewood/blob/938a9cdb6aa5de1c82a12ac9dc226a48026a76e2/fwdctl/src/check.rs#L11

Reply:
OH, sorry. I misunderstood the diff between changes! The dependency is still here.